src/applications/static-pages/sharable-links/createSharableLink.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import { Provider } from 'react-redux';
+
+export default function createSharableLink(store, widgetType) {
+  const sharableLinks = document.querySelectorAll(
+    `[data-widget-type="${widgetType}"]`,
+  );
+  if (sharableLinks.length > 0) {
+    import(/* webpackChunkName: "sharableLink" */
+    './sharableLink').then(module => {
+      const SharableLink = module.default;
```

Comment:
@ncksllvn let me know if you have any suggestions for how to optimize this iteration. I tried using `Array.from` but ran into issues

Reply:
Hm, I'm not sure. Since there may be a lot of instances, we might want to evaluate the performance of using a Provider and `ReactDOM.render` for each one